packages/cli/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to the Zowe CLI package will be documented in this file.
 
+
+## `6.37.8`
```

Comment:
Please change the header to `## Recent Changes`. Our deployment automation will update this to a version number later.